index.bs

```diff
@@ -1689,12 +1695,19 @@ An {{XRPose}} describes a position and orientation in space relative to an {{XRS
 <pre class="idl">
 [SecureContext, Exposed=Window] interface XRPose {
   [SameObject] readonly attribute XRRigidTransform transform;
+  [SameObject] readonly attribute DOMPointReadOnly? linearVelocity;
+  [SameObject] readonly attribute DOMPointReadOnly? angularVelocity;
+
   readonly attribute boolean emulatedPosition;
 };
 </pre>
 
 The <dfn attribute for="XRPose">transform</dfn> attribute describes the position and orientation relative to the base {{XRSpace}}.
 
+The <dfn attribute for="XRPose">linearVelocity</dfn> attribute describes the linear velocity in meters per second relative to the base {{XRSpace}}. If the user agent can't populate this, it's allowed to return <code>null</code>.
+
+The <dfn attribute for="XRPose">angularVelocity</dfn> attribute describes the angular velocity in meters per second relative to the base {{XRSpace}}. If the user agent can't populate this, it's allowed to return <code>null</code>.
```

Comment:
Angular velocity isn't measured in meters per second. It seems like OpenXR reports this in radians per second, which seems fairly sensible.